### ISSUE TITLE
Failure to scan Collections in SKSE/Plugins

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#201]
Nexus followup, used to just CTD but now handles exceptions

> 2020-08-26 12:30:42.965     info   2804 *** LOAD *** Build Collections
> 2020-08-26 12:30:42.971     info   2804 JSON Schema C:\Program Files (x86)\Steam\steamapps\common\Skyrim Special Edition\Data\SKSE\Plugins\SHSE.SchemaCollections.json parsed and validated
> 2020-08-26 12:30:42.973    error   2804 Collection Definitions not loadable: is the file target at C:\Program Files (x86)\Steam\steamapps\common\Skyrim Special Edition\Data\SKSE\Plugins\ inside 'Program Files' or another elevated-privilege location? Error:
> No mapping for the Unicode character exists in the target multi-byte code page.

**Comment by SteveTownsend** [comment]
Try the work round suggested at https://stackoverflow.com/questions/53821593/c-recursive-directory-iterator-miss-some-files

> You can however set your locale to UTF-16LE to remedy this. It works with /std:c++17 and the standard <filesystem> header
```
```
More support for this approach here: https://developercommunity.visualstudio.com/content/problem/721120/stdfilesystempathgeneric-string-throws-an-exceptio.html

> generic_string() does not return Unicode. It returns characters in the current active code page. The design intent is that you can take the result and call fopen() with it and open the file to which the path refers.

**Comment by SteveTownsend** [comment]
Tested OK locally using user file with Cyrillic characaters in its name. Fixed in v2.9.6.0
```
2020-09-04 08:41:30.199     info   6072 *** LOAD *** Build Collections
2020-09-04 08:41:30.218     info   6072 JSON Schema J:\SteamLibrary\steamapps\common\Skyrim Special Edition\Data\SKSE\Plugins\SHSE.SchemaCollections.json parsed and validated
2020-09-04 08:41:30.221     info   6072 Skip CBPCollisionConfig_Ригмор.txt, does not match Collections filename pattern
```
